backend/app/services/llm_judge.py

```
@@ -4,151 +4,77 @@
-
-    def evaluate(self):
-        trace = self.load_latest_trace()
-        results = {
-            "usefulness": self.evaluate_one("usefulness", trace),
-            "groundedness": self.evaluate_one("groundedness", trace),
-            "efficiency": self.evaluate_one("efficiency", trace)
-        }
-        with OUTPUT_FILE.open("w", encoding="utf-8") as f:
-            json.dump(results, f, ensure_ascii=False, indent=2)
+        raw_text = response.content if hasattr(response, "content") else str(response)
+        clean_text = raw_text.strip().lstrip("```json").rstrip("```").strip()
```

Ruff (0.15.12)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)